kp: 40.0
ki: 0.42 # sem correntes
#ki: 5.0 #com correntes
kff: 48.0
kff_lin_drag: 0.0
kff_quad_drag: 0.0
max_ref:  1.2  # m/s
min_ref: -1.2  # m/s
max_err:  0.5  # m/s
min_err: -0.5  # m/s  
max_out:  81.0 # N
min_out: -81.0
debug: true

m: 11.5 #kg
Yu_dot: -25.952
